backend/controller/controller.go

```diff
@@ -226,6 +227,10 @@ func New(ctx context.Context, pool *pgxpool.Pool, config Config, runnerScaling s
 	}
 	config.SetDefaults()
 
+	if err := observability.InitControllerObservability(); err != nil {
+		log.FromContext(ctx).Warnf("failed to initialize controller observability: %v", err)
+	}
```

Comment:
This works for now, but we might want to consider combining this and  the `observability.Init()` that's currently in `ftl-controller/main` `ftl-runner/main` and `cmd_serve` files now. We can iterate on that in the future.

Reply:
will combine in a fast follow change that completes the fsm metric instrumentation